fdbclient/FDBTypes.h

```diff
@@ -706,6 +706,7 @@ struct RangeResultRef : VectorRef<KeyValueRef> {
 		       " readToBegin:" + std::to_string(readToBegin) + " readThroughEnd:" + std::to_string(readThroughEnd);
 	}
 };
+using RangeResult = Standalone<RangeResultRef>;
```

Comment:
I understand putting this declaration after the `RangeResultRef` struct, but the rest of the typedefs (`Key`, `Value`, ...) are clustered together. Do you prefer to leave this here or move it to be with the other similar typedefs?

Reply:
Good idea, I've moved this now.